src/co/edu/uptc/controller/RoleController.java

```diff
@@ -0,0 +1,60 @@
+package co.edu.uptc.controller;
+
+import java.util.ArrayList;
+
+import co.edu.uptc.model.Role;
+import co.edu.uptc.persistence.Persistence;
+
+public class RoleController {
+
+    private Persistence<Role> persistence;
+
+    public RoleController() {
+    }
+
+    public RoleController(Persistence<Role> persistence) {
+        this.persistence = persistence;
+    }
+
+    public boolean add(Role role) {
+        return this.persistence.persist(role);
+    }
+
+    public boolean delete(int id) {
+        return this.persistence.erase(id);
+    }
+
+    public Role get(int id) {
+        return this.persistence.obtainById(id);
+    }
+
+    public Role update(int option, int roleId, int newId, String name) {
```

Comment:
Generalmente para actualizar es mejor mandar una clase de lo que se quiere actualizar, suponga.

```suggestion
    public Role update(int id, Role newRole)
```
 En este caso es necesario saber cuál es el role que quiero modificar y solamente sería enviar a lo que quiero cambiar